src/client/features/interactions/index.js

```diff
@@ -15,7 +15,7 @@ class Interactions extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      cy: make_cytoscape({ headless: true, stylesheet: interactionsStylesheet }),
+      cy: make_cytoscape({ headless: true, stylesheet: interactionsStylesheet, edgeTooltips:true }),
```

Comment:
Minor nitpick but this should be renamed to communicate that this is a boolean. It could be interpreted that the param edgeToolTips are actual tooltips being passed in.